core/admin/mailu/internal/views/auth.py

```diff
@@ -31,7 +31,7 @@ def nginx_authentication():
     for key, value in headers.items():
         response.headers[key] = str(value)
     is_valid_user = False
-    is_from_webmail = headers['Auth-Port'] in ['10143', '10025']
+    is_from_webmail = headers.get('Auth-Port') in ['10143', '10025']
```

Comment:
I'll try to test this on my system, but i'm looking at line 15 and it appears that this key is already accessed without errors, unless python is automatically ignoring the missing key because it's in a condition?